src/avram/migrator/migration.cr

```diff
@@ -62,7 +62,7 @@ abstract class Avram::Migrator::Migration::V1
 
   def migrated?
     DB.open(Avram::Migrator::Runner.database_url) do |db|
-      db.query_one? "SELECT id FROM migrations WHERE version = $1", version, as: Int32
+      db.query_one? "SELECT id FROM migrations WHERE version = $1", version, as: Int
```

Comment:
I think you're right.. This should probably be `Int64`. I'm not sure how well supported using the generic `Int` is. I've had issues with it a few times. Let's try `Int64` and see if that works out.

Reply:
Int64 seems to cause some problems for me on postgres.  I will have to look into it more.

Reply:
I'm actually surprised that crystal-db didn't blow up on that before. I wonder if that's an issue? 🤔  Let me know what you find out.